samples/abn-sample/frontend/go/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Get Iter8
+FROM golang:1.19-buster as builder
```

Comment:
At least between the two Docker images, we can make it consistent.

Reply:
done; both moved to 1.19